backend/src/managers/NotificationManager.ts

```diff
@@ -193,7 +195,9 @@ export default class NotificationManager {
     this.userCache.deleteUserStatsCache(forUserId)
 
     // send push in background
-    this.sendWebPush(forUserId, notification).then().catch()
+    this.sendWebPush(forUserId, notification)
+      .then()
+      .catch((err) => this.logger.error(err))
```

Comment:
maybe add a bit more info?

```suggestion
      .catch((err) => this.logger.error('Failed to send web push', { forUserId, error })
```

Reply:
so, what's the resolution?